docs/Architecture.md

```diff
@@ -103,10 +103,24 @@ For the visual part, react-admin builds upon MUI, which is the implementation of
 
 We spend a great deal of time refining the UI to make it as intuitive as possible. We pay attention to small alignment glitches, screen flashes, and color inconsistencies. We iterate with every customer feedback, to remove visual and animation problems that occur in real-life applications.
 
-With theme customization, any developer can make an ugly react-admin app, but we do our best to make it not too shabby by default. React-admin produces a user interface that is voluntarily bland by default because we want to emphasize content rather than chrome.
+React-admin produces a user interface that is voluntarily bland by default because we want to emphasize content rather than chrome.
 
 ![Sort Button](./img/sort-button.gif)
 
+As for the developer experience, react-admin is constantly evolving to find the sweet spot between an intiutive API, power user features, not too much magic, and just enough documentation. The core team are the first testers of react-admin, and pay attention to the productivity, debuggability, discoverability, performance, and robustness of all the hooks and compnents.
+
+## Built On The Shoulders Of Giants
+
+Many excellent open-source libraries already address partial requirements of B2B apps: data fetching, forms, UI components, testing, etc.
+
+Rather than reinventing the wheel, react-admin uses the the best tools in each category (in terms of features, developer experience, active maintenance, documentatino, user base), and provides a glue around these libraries.
```

Comment:
```suggestion
Rather than reinventing the wheel, react-admin uses the the best tools in each category (in terms of features, developer experience, active maintenance, documentation, user base), and provides a glue around these libraries.
```